action.yml

```diff
@@ -7,7 +7,7 @@ inputs:
     default: "."
 runs:
   using: docker
-  image: docker://ghcr.io/lunarmodules/luacheck:v1.1.2
+  image: Dockerfile
```

Comment:
Can you rebase to pull out the commit that changes this line please?

Reply:
done :+1: